polyfill/test/usercalendar.mjs

```diff
@@ -224,29 +224,61 @@ describe('Userland calendar', () => {
   });
   describe('Trivial protocol implementation', () => {
     // For the purposes of testing, a nonsensical calendar that has 10-month
-    // years and 10-day months, and the year zero is at the Unix epoch
-    function decimalToISO(year, month, day, overflow) {
+    // years, 10-day months, 864-hour days, 10-minute hours and 10-second
+    // minutes, and the year zero is at the Unix epoch.
+    function decimalToISO(year, month, day, hour = 0, minute = 0, second = 0, overflow = 'constrain') {
       if (overflow === 'constrain') {
         if (month < 1) month = 1;
         if (month > 10) month = 10;
         if (day < 1) day = 1;
         if (day > 10) day = 10;
+        if (hour < 1) hour = 1;
+        if (hour > 864) hour = 864;
```

Comment:
```suggestion
        if (hour < 0) hour = 0;
        if (hour > 863) hour = 863;
```

Reply:
Fixed reject, but somehow missed this one! Thanks.